src/components/organisms/navigation-bar/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from "react-router-dom";
+import classnames from "classnames";
+import styles from "./index.module.scss";
+
+export function NavigationBar() {
+  return (
+    <div>
+      <Link className={classnames(styles.navBtn)} to={"/pro"}>
```

Comment:
@1gnoramus look here. Now the css classes need to be imported and used like this